### PR TITLE
test: add e2e test for column level security

### PR DESCRIPTION
**Pull request by rephus**
Closes: #6653

### Description:

[Screencast from 11-08-23 11:22:15.webm](https://github.com/lightdash/lightdash/assets/1983672/96c21c76-4b59-4a82-999f-c99641252e28)

### CODE REVIEW

File: packages/e2e/cypress/e2e/userAttributes.cy.ts [hunk 1]
```diff
@@ -81,3 +81,88 @@ describe('User attributes', () => {
...
+
+    it('Should see last_name attribute', () => {
+        cy.visit(`/projects/${SEED_PROJECT.project_uuid}/tables`);
+
+        cy.findByText('Users').click();
+        cy.findByText('Last name').click();
+
+        // run query
+        cy.get('button').contains('Run query').click();
+        cy.contains('W.');
```
Comment (joaoviana) [inline packages/e2e/cypress/e2e/userAttributes.cy.ts:151]: I don't know what this is 😅

**Review by joaoviana — APPROVED** [review]
looks great! one q but non-blocking

File: packages/e2e/cypress/e2e/userAttributes.cy.ts (hunk 1 above)
Comment (rephus) [reply packages/e2e/cypress/e2e/userAttributes.cy.ts:151]: This means the user with id 30 should be able to see its own last_name: 'W.'

Comment (joaoviana) [reply packages/e2e/cypress/e2e/userAttributes.cy.ts:151]: thanks!

**Pull request merged**